__tests__/app.test.js

```diff
@@ -213,6 +213,24 @@ describe("DELETE /api/comments/:comment_id", () => {
   });
 });
 
+describe("GET /api/users", () => {
+  test("200 Get all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.users.forEach((user) => {
+          expect(Object.keys(user)).toHaveLength(3);
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
```

Comment:
Nice test, but you should have an assertion to ensure the array isnt empty before this forEach(...)